decisions/2020/04/28-ui-forms-refactor

```diff
@@ -0,0 +1,31 @@
+Date: 2020-04-28\
+Attendees: Ismay, Jan-Gerke, Viktor
+
+# Agenda
+
+Simplification and consistency of the form controls, e.g. Radio, Checkbox, Switch, Field, *Field-components.
+
+Touched on future changes we want to explore for ui@6 as well.
+
+# @dhis2/ui-core
+
+- Drop the Field component
+- Re-implement as a generic form field
```

Comment:
So to clarify for myself, what we mean here is basically; change the `<Field>` component from just something that provides spacing, to a component that wraps a form 'field' (for lack of a better word) and adds helpText, validationText, etc. Is that right?

Reply:
So that you can do:

```jsx
<Field helpText="Help text" {// etc.}>
  <Input {...} />
</Field>
```

Reply:
That's correct: https://github.com/dhis2/ui/pull/46/files#diff-0556b6ad0bddb321a9ddd41f13279cb6R31-R48

Reply:
Ok cool, thx 👍